src/wallet/test/wallet_tests.cpp

```diff
@@ -426,7 +426,7 @@ BOOST_AUTO_TEST_CASE(LoadReceiveRequests)
 
 // Test some watch-only LegacyScriptPubKeyMan methods by the procedure of loading (LoadWatchOnly),
 // checking (HaveWatchOnly), getting (GetWatchPubKey) and removing (RemoveWatchOnly) a
-// given PubKey, resp. its corresponding P2PK Script. Results of the the impact on
+// given PubKey, resp. its corresponding P2PK Script. Results of the impact on
```

Comment:
I guess someone really likes the band? anyway, ACK but could you also fix this one while here

```
$ test/lint/lint-spelling.sh 
src/crypto/sha256_arm_shani.cpp:65: implemenation ==> implementation
```

Reply:
Done! https://github.com/bitcoin/bitcoin/pull/24359/commits/9c2bb5a2b499663c386aa9641baa85558aaa25ff